src/cleanupEquation.js

```diff
@@ -0,0 +1,20 @@
+import closeParens from "./closeParens";
+import addMultiplier from "./addMultiplier";
+
+const getLastChar = (from) => {
```

Comment:
@grimxreaper  usually better to name parameters so the code inside reads well. You aren't getting a "from", you're getting something like "characters" and slicing the last one off. Though "getLastChar from" does read well and I appreciate you thinking about good names!

Reply:
cleanupEquation.js no longer exists, also "from" is now "char" :)

Reply:
great! you might consider changing `char` to `chars` so you don't give the impression you're only expecting a single char to be passed in.